app/decorators/physical_rack_decorator.rb

```diff
@@ -0,0 +1,24 @@
+class PhysicalRackDecorator < MiqDecorator
+  def fileicon
+    # TODO: change it once we have a file icon for a rack
+    nil
+  end
+
+  def self.fonticon
+    'pficon pficon-enterprise'
+  end
+
+  def quadicon(_n = nil)
+    icon = {
+      :top_left => {:text => (physical_servers ? physical_servers.size : 0)}
+    }
+    icon[:middle] = { :fileicon => '100/shield.png' } if get_policies.present?
+    icon
+  end
+
+  def single_quad
+    {
+      :fileicon => fileicon
```

Comment:
What kind of icon this fileicon will be? Is it a vendor-related logo (then it should be SVG) or a simple monochromatic icon (it should be fonticon then). If you don't have a fonticon for this, we can and add it into [our own icon set](https://github.com/ManageIQ/font-fabulous) from an SVG file. We're now trying to get rid of the `png` files and prefer fonticons and SVG-based fileicons. ping @epwinchell

Reply:
Right, we don't have anything to use as a single_quad, and we don't have enough info to use the four quadicon quadrants, what is your suggestion here? What I was thinking was to use a quadicon util this is decided, or we have more info do fill the other quadrants.

Reply:
Well, if you can explain me (here or on gitter) what exactly the physical rack is, I might come up with some quadrant ideas for you.

Reply:
Discussing aside with @skateman, I decided to support only single_quads for the moment, because Racks don't support policies yet. Do you guys agree?